pytorch_lightning/core/datamodule.py

```diff
@@ -398,3 +325,62 @@ def test_dataloader():
         if test_dataset is not None:
             datamodule.test_dataloader = test_dataloader
         return datamodule
+
+    def __new__(cls, *args: Any, **kwargs: Any) -> 'LightningDataModule':
+        obj = super(LightningDataModule, cls).__new__(cls)
```

Comment:
do you need to specify the arguments to `super` here? Couldn't you just call `super()`?

Reply:
Doesn't seem like it:

```suggestion
        obj = super().__new__(cls)
```